assets/css/sass/partials/_subheader.scss

```diff
@@ -187,31 +187,32 @@
         span {
             font-weight: bold;
         }
-
-        .half{
-            display: inline;
-            width: inherit;
+        .stats-list {
+            display: block;
+            width: 100%;
             padding: 0px;
 
-            label{
-                float: right;
+            .exportBtn {
+                margin-left: 10px;
+                margin-top: -12px;
+                width: 135px;
+                display: inline-block;
+            }
+            #tree-and-planting-site-counts {
+                max-width: calc(100% - 305px);
+                display: inline-block;
```

Comment:
Honestly I arrived at most of these pixel amounts via trial-and-error, not from adding up the widths and margins of various other things. I added a comment to this effect in https://github.com/OpenTreeMap/otm-core/pull/2787/commits/e97505c17612282f7555c11a6cf045703589080e